tests/integration/test_uninstall.py

```diff
@@ -90,11 +90,8 @@ def test_uninstall_all_local_files(pipenv_instance_private_pypi, testsroot):
         c = p.pipenv(f"install {file_uri}")
         assert c.returncode == 0
         c = p.pipenv("uninstall --all")
-        assert c.returncode == 0
-        assert "tablib" in c.stdout
-        # Uninstall --all is not supposed to remove things from the pipfile
-        # Note that it didn't before, but that instead local filenames showed as hashes
-        assert "tablib" in p.pipfile["packages"]
+        assert "tablib" not in p.pipfile["packages"]
+        assert "tablib" not in p.lockfile["default"]
```

Comment:
This is the change that allowed part of [this regression](https://github.com/pypa/pipenv/issues/6200) to sneak through.